content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mgm-onboarding.md

```diff
@@ -74,21 +74,6 @@ Set the values of variables for use in later commands:
    {{% /tab %}}
    {{< /tabs >}}
 
-4. Set the path to your local clone of `corda-runtime-os`:
```

Comment:
Sorry - just realised I didn't mention specifically which files I was talking about in regards to this. I think this is still needed to the mgm-onboarding because it's referenced in one place, but it's safe to remove from dynamic-onboarding